backend/src/packages/orders/order.controller.ts

```diff
@@ -410,11 +415,16 @@ class OrderController extends Controller {
       user: UserEntityObjectWithGroupT | null;
     }>,
   ): Promise<ApiHandlerResponse<OrderResponseDto>> {
+    const customerPhone = options.body.customerPhone ?? options.user?.phone;
+    const customerName = options.body.customerName ?? options.user?.firstName;
+
     return {
       status: HttpCode.OK,
       payload: await this.orderService.create({
         ...options.body,
         userId: options.user?.id ?? null,
+        customerPhone: customerPhone ?? null,
+        customerName: customerName ?? null,
```

Comment:
I think it would be a little cleaner that way.
```suggestion
    const userId = options.user?.id ?? null;
    const customerPhone = options.body.customerPhone ?? options.user?.phone ?? null;
    const customerName = options.body.customerName ?? options.user?.firstName ?? null;

    return {
      status: HttpCode.OK,
      payload: await this.orderService.create({
        ...options.body,
        userId,
        customerPhone,
        customerName,
```